### PR TITLE
update zh-trans content/zh/docs/reference/kubectl/conventions.md

### PR DESCRIPTION
**Pull request by tanjunchen**
ref:https://github.com/k8smeetup/website-tasks/issues/3688
/cc @tengqm

### CODE REVIEW

File: content/zh/docs/reference/kubectl/conventions.md [hunk 1]
```diff
@@ -82,82 +82,72 @@ You can create the following resources using `kubectl run` with the `--generator
...
+| Pod                                  | v1                 | `kubectl run --generator=run-pod/v1`              |
+| ReplicationController _(deprecated)_ | v1                 | `kubectl run --generator=run/v1`                  |
+| Deployment _(deprecated)_            | extensions/v1beta1 | `kubectl run --generator=deployment/v1beta1`      |
+| Deployment _(deprecated)_            | apps/v1beta1       | `kubectl run --generator=deployment/apps.v1beta1` |
+| Job _(deprecated)_                   | batch/v1           | `kubectl run --generator=job/v1`                  |
+| CronJob _(deprecated)_               | batch/v2alpha1     | `kubectl run --generator=cronjob/v2alpha1`        |
+| CronJob _(deprecated)_               | batch/v1beta1      | `kubectl run --generator=cronjob/v1beta1`         |
+{{< /table >}}
+-->
+{{< table caption="使用 kubectl run 创建资源" >}}
```
Comment (tengqm) [inline content/zh/docs/reference/kubectl/conventions.md]:
```suggestion
{{< table caption="可以使用 kubectl run 创建的资源" >}}
```

File: content/zh/docs/reference/kubectl/conventions.md [hunk 2]
```diff
@@ -82,82 +82,72 @@ You can create the following resources using `kubectl run` with the `--generator
...
-and `--generator=run/v1`, a Replication Controller is created.
-This enables you to pin to a specific behavior with the generator,
-even when the default generator is changed later.
--->
-只有在未指定任何参数时，这些参数才使用默认生成器。
-这意味着，当您将 `--generator` 与其他参数组合时，随后指定的生成器不会更改。
-例如，在集群版本 v1.4 中，如果最初指定了 `--restart=always`，则会创建 Deployment；如果后来指定了 `--restart=always` 和 `--generator=run/v1`，则会创建 Replication Controller。
-这使您能够将生成器固定到特定的行为，即使在以后更改默认生成器时也是如此。
-{{< /note >}}
+{{< table caption="kubectl 运行参数及其匹配的资源" >}}
```
Comment (tengqm) [inline content/zh/docs/reference/kubectl/conventions.md]:
```suggestion
{{< table caption="kubectl run 参数及其对应的资源" >}}
```

**Comment by tanjunchen** [comment]
/cc @tengqm
done for update
/cc @xichengliudui
/cc @SataQiu

**Pull request merged**